sw/device/rom_exts/manifest.md

```diff
@@ -0,0 +1,67 @@
+# ROM_EXT Manifest Format
+
+Based on the format outlined in [boot.md](./boot)
+
+```
+ 0                   1                   2                   3
+ 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+ ───────────
+|                           Identifier                          |
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+|                                                               |
++                                                               +
+|                                                               |
++                           Signature                           +
+|                                                               |  Unsigned
++                          (Size TBD)                           +    Area
+|                                                               |
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+ ───────────
+|             Length            |      Signature Algorithm      |
```

Comment:
Would it make sense to make length 32bit, marking unused bits as reserved? Potential future proofing.

Reply:
Let's make it 32-bit, but not mark unused bits as reserved.